apps/aecore/src/aec_accounts.erl

```diff
@@ -33,15 +36,18 @@
 
 -define(ACCOUNT_VSN_1, 1). %% {Nonce, Balance}
 -define(ACCOUNT_VSN_2, 2). %% {Flags, Nonce, Balance, GA_Contract, GA_AuthFun}
+-define(ACCOUNT_VSN_3, 3). %% {Flags, Nonce, Balance}
 -define(ACCOUNT_TYPE, account).
 
 -type fun_hash() :: <<_:256>>.
+-type flag()  :: non_payable.
+-type flags() :: [flag()].
 
 -record(account, {
           id          :: aeser_id:id(),
           balance = 0 :: non_neg_integer(),
           nonce = 0   :: non_neg_integer(),
-          flags = 0   :: non_neg_integer(),  %% NOTE: not yet in use
+          flags = 0   :: non_neg_integer(),  %% non_payable | ...
```

Comment:
for later... we might want to document how these flags are encoded to integers as a comment above the record